packages/terra-slide-panel/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Corrected slide panel's dom ordering
```

Comment:
I think it is worth mentioning that the DOM ordering is being flexed based upon the `panelPosition` prop in the CHANGELOG.

Reply:
Corrected the CHANGELOG.md file to be more descriptive.